README.md

```diff
@@ -1,2 +1,118 @@
-# aspose-imaging-cloud-javascript
-JavaScript SDK for communicating with the Aspose.Imaging Cloud API
+# Aspose.Imaging Cloud JavaScript SDK
+[Aspose.Imaging Cloud](https://products.aspose.cloud/imaging) is a true [REST API](https://apireference.aspose.cloud/imaging/) that enables you to perform a wide range of image processing operations including creation, manipulation and conversion in the cloud, with zero initial costs. Our Cloud SDKs are wrappers around REST API in various programming languages, allowing you to process images in language of your choice quickly and easily, gaining all benefits of strong types and IDE highlights. 
+
+This repository contains [Aspose.Imaging Cloud JavaScript SDK](https://products.aspose.cloud/imaging/javascript) source code which is written in TypeScript. This SDK allows you to work with Aspose.Imaging Cloud REST APIs in your web and applications quickly and easily, with zero initial cost.
+
```

Comment:
Please remove the reference to https://products.aspose.cloud/imaging/javascript till we add this landing page.